src/main/java/ca/unknown/bot/app/Main.java

```diff
@@ -1,5 +1,7 @@
 package ca.unknown.bot.app;
 
+import ca.unknown.bot.use_cases.*;
```

Comment:
might not need this as we are importing specific classes from the use_cases package in the following lines

Reply:
true thank you it was unused